src/components/AboutContainer/AboutContentContainer/AboutContentPages/GetInvolved.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import GetInvolved from './GetInvolved';
+
+('use strict');
+
+it('renders the page and shows the headings', () => {
+  render(<GetInvolved />);
+  expect(
+    screen.getByText('We are proudly volunteer-driven')
+  ).toBeInTheDocument();
+  expect(screen.getByText('Donate')).toBeInTheDocument();
+  expect(screen.getByText('Grow with us')).toBeInTheDocument();
+  expect(screen.getByText('Mentor')).toBeInTheDocument();
+  expect(screen.getByText('Volunteer')).toBeInTheDocument();
+});
+
+const Button = ({ onClick, children }) => (
+  <button onClick={onClick}>{children}</button>
+);
+
+it('checks that the Make a Donation button is clickable', () => {
+  const handleClick = jest.fn();
+  render(<Button onClick={handleClick}>MAKE A DONATION</Button>);
+  fireEvent.click(screen.getByText(/make a donation/i));
+  expect(handleClick).toHaveBeenCalledTimes(1);
+});
```

Comment:
this is not actually testing the button that is in the component; what we're doing here is rendering a button in jsdom and clicking it; not the same button that is in the component

Reply:
will come back to improve this later